clang/test/CXX/drs/cwg25xx.cpp

```diff
@@ -92,6 +92,26 @@ using ::cwg2521::operator""_div;
 #endif
 } // namespace cwg2521
 
+namespace cwg2547 { // cwg2547: 20
+#if __cplusplus >= 202302L
+struct S; // since-cxx23-note 3 {{forward declaration of 'cwg2547::S'}}
+bool operator==(S, S) = default;  // error: S is not complete
+// since-cxx23-error@-1 2 {{variable has incomplete type 'S'}}
```

Comment:
```suggestion
// since-cxx23-error@-1 {{variable has incomplete type 'S'}}
// since-cxx23-error@-2 {{variable has incomplete type 'S'}}
```
We don't use diagnostic quantifiers in DR test suite